README.md

```diff
@@ -369,7 +369,6 @@ Other Style Guides
     const itemsCopy = [...items];
     ```
 
-  <a name="arrays--from">
```

Comment:
This needs to stay to preserve the link; instead, can you close the tag with `</a>`?